androidTEst/androidTEst/Screens/AllFilesScreen.cs

```diff
@@ -0,0 +1,116 @@
+﻿using androidTEst.Extension;
+using Aquality.Appium.Mobile.Elements.Interfaces;
+using Aquality.Appium.Mobile.Screens;
+using OpenQA.Selenium;
+
+namespace Aquality.Appium.Mobile.Template.Screens.Login
+{
+    public class AllFilesScreen : Screen
+    {
+        private readonly IButton AddNewFileButt = ElementFactory.GetButton(By.Id("com.nextcloud.client:id/fab_main"), "AddNewFileButt");
+        private readonly IButton MenuButt = ElementFactory.GetButton(By.Id("com.nextcloud.client:id/menu_button"), "MenuButt");
+        private readonly IButton AllFilesButt = ElementFactory.GetButton(By.Id("com.nextcloud.client:id/sort_list_button_group"), "AllFilesButt");
+
+        // regardless of the language of the system/application, the text "текстовый документ" doesn't change. (see ./Screenshots)
+        private readonly IButton CreateDoc = ElementFactory.GetButton(By.XPath("//android.widget.TextView[@resource-id= 'com.nextcloud.client:id/creator_name' and contains(@text, 'текстовый документ')]"), "CreateDoc");
+        private readonly ITextBox TextFileName = ElementFactory.GetTextBox(By.Id("com.nextcloud.client:id/filename"), "TextFileName");
```

Comment:
```suggestion
        private readonly ITextBox FileNameTextBox = ElementFactory.GetTextBox(By.Id("com.nextcloud.client:id/filename"), "TextFileName");
```
оконо вввода имени файла тоже вполне себе может быть самостоятельной формой, можешь описать их как вложенные классы, если они могут использоваться только на этой странице, но все же это отдельная сущность